recipes-kernel/kexec/kexec-tools/kdump.conf

```diff
@@ -0,0 +1,19 @@
+#the kdump kernel version string.
+KDUMP_KVER="`uname -r`"
+
+#this will be passed to the kdump kernel as kdump kernel command line
+#only go to runlevel 3 with nonet nousb nosound nomodeset to make boot faster
+#blacklist nikal so no NI driver module would be loaded
+KDUMP_CMDLINE="`cat /proc/cmdline` 3 nonet nousb nosound nomodeset module_blacklist=nikal"
```

Comment:
nonet nousb nosound and nomodeset did not actually do anything @gratian ?

Reply:
`nonet` and `nosound` don't exist in https://www.kernel.org/doc/html/latest/admin-guide/kernel-parameters.html. `nousb` might be `usbcore.nousb`. `nomodeset` should still be operational-- if you have that on you _should_ only get EFI framebuffer instead of i915, I think? But it's possible that those others might have been removed in earlier kernel versions.

Reply:
Ah, updated documentation! Thanks, drop `nousb` ,`nonet`, and `nosound`. Add `usbcore.nousb` and keep `nomodeset`.